### PR TITLE
enhance(logs): refactor and clean up logs controller and style

### PR DESCRIPTION
**Pull request by davidvader**
3 changes
- refactor existing logs style, cleaner structure, removed redundant css
- made step skipped/errors more readable, moved red font to error label only
- added a message for empty logs and empty error (null)

Screen Shot 2020-10-06 at 3 41 44 PM

### CODE REVIEW

File: src/elm/Pages/Build/View.elm [hunk 1]
```diff
@@ -603,36 +604,50 @@ stepFollowButton stepNumber following =
...
+    div [ class "message" ]
+        [ Util.smallLoaderWithText "loading..." ]
+
+
+{-| emptyLogs : renders message for empty logs
+-}
+emptyLogs : Html msg
+emptyLogs =
+    div [ class "message" ]
+        [ text "The build has not written logs to this step yet." ]
```
Comment (kneal) [inline src/elm/Pages/Build/View.elm:631]: Really minor but I noticed all the other messages are all in lowercase. Would it make sense for this one as well?

Comment (davidvader) [reply src/elm/Pages/Build/View.elm:631]: great point, I agree @kneal 👍

Comment (davidvader) [reply src/elm/Pages/Build/View.elm]: updated! 🚀

**Review by wass3r — APPROVED** [review]

**Review by kneal — APPROVED** [review]
LGTM 🐬

**Pull request merged**